POLICY.md

```diff
@@ -55,6 +58,7 @@ This document describes rules that are in effect for this repository, meant for
 4.	Pull requests that address an outstanding issue, particularly an issue deemed to be severe, should be given priority
 5.	If a PR is accepted, then it should undergo review and updated based on the feedback provided, then merged
 6.   Pull requests that don't meet the above will be denied and closed
+7.   Pull request accepted and that modify something in the documentation will be marked with the label DOC, for to be able to find it easily, review it and translate it.
```

Comment:
There is an existing label, `component: documentation`, which can be used for issues and PRs. Can we use it instead of introducing a new label?